### ISSUE TITLE
Bad references in documentation for VkBlitImageInfo2KHR

### ISSUE THREAD

**Issue by expipiplus1** [KhronosGroup/Vulkan-Docs#1407]
see here: https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkBlitImageInfo2KHR.html

The valid usage statements are full of `{refpage}` where it should read `VkBlitImageInfo2KHR`. For example:

```
[[VUID-{refpage}-pRegions-00215]] VUID-{refpage}-pRegions-00215
The source region specified by each element of pRegions must be a region that is contained within srcImage
```

**Comment by oddhack** [comment]
Thanks. That's an oversight in the markup, should be corrected in the next update.

**Comment by oddhack** [comment]
This should be fixed in the 1.2.163 spec update.

**Comment by expipiplus1** [comment]
Thanks!
